kotlin-example/src/test/kotlin/org/onflow/examples/kotlin/addKey/AddAccountKeyExampleTest.kt

```diff
@@ -0,0 +1,40 @@
+package org.onflow.examples.kotlin.addKey
+
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.onflow.examples.kotlin.AccessAPIConnector
+import org.onflow.flow.common.test.*
+import org.onflow.flow.sdk.*
+
+@FlowEmulatorProjectTest(flowJsonLocation = "../flow/flow.json")
+internal class AddAccountKeyExampleTest {
+    @FlowTestAccount
+    lateinit var testAccount: TestAccount
+
+    @FlowTestClient
+    lateinit var accessAPI: FlowAccessApi
+
+    private lateinit var accessAPIConnector: AccessAPIConnector
+    private lateinit var connector: AddAccountKeyExample
+
+    @BeforeEach
+    fun setup() {
+        connector = AddAccountKeyExample(testAccount.privateKey, accessAPI)
+        accessAPIConnector = AccessAPIConnector(testAccount.privateKey, accessAPI)
+    }
+
+    @Test
+    fun `Can add key to account`() {
+        val initialKeys = accessAPIConnector.getAccount(testAccount.flowAddress)
+        Assertions.assertEquals(2, initialKeys.keys.size, "The account should initially have 2 keys")
+
+        val txResult = connector.addKeyToAccount(testAccount.flowAddress)
+
+        Assertions.assertNotNull(txResult, "Transaction result should not be null")
+        Assertions.assertTrue(txResult.status == FlowTransactionStatus.SEALED, "Transaction should be sealed")
+
+        val updatedAccount = accessAPIConnector.getAccount(testAccount.flowAddress)
+        Assertions.assertEquals(3, updatedAccount.keys.size, "The account should now have 3 keys")
```

Comment:
I know this is only an example, but it would be good to add an extra check that the 3rd key is the key we expect to be added. 
Adding keys to an account is particularly a risky operation since it "gives" the power of controlling an account to the new key. Adding the wrong key can lead to losing all the assets of the account. Therefore, it would be good if the example promotes how to check that the operation went well:
 - we could make  `addKeyToAccount` accept an input public key `pk` (in the form of a flow account key for instance)
 - `pk` is generated within the test, and is passed to `addKeyToAccount` 
 - we add a final test that `updatedAccount.keys[3]` is `pk` (checking the hex value only is enough)